cypress/e2e/nodes/PreviewOptions.spec.js

```diff
@@ -42,8 +42,8 @@ describe('Preview Options', function() {
 	})
 
 	it('should Remove link', function() {
-		cy.get('.paragraph-content').should('have.text', 'nextcloud.com')
+		cy.get('p').should('have.text', 'nextcloud.com')
```

Comment:
hmm... That selector looks super generic. Doesn't it match any `p` element on the page?

Reply:
Maybe we can bring back the `paragraph-content` class - the other `p` selectors in css are also quite generic.

Reply:
I changed it to `p > a` now, which also fixes the test.